src/engine/SpatialJoinAlgorithms.h

```diff
@@ -193,4 +218,13 @@ class SpatialJoinAlgorithms {
 
   // return whether one of the poles is being touched
   std::array<bool, 2> isAPoleTouched(const double& latitude) const;
+
+  // number of times the parsing of a geometry failed. For now this is only used
+  // to print the warning once, but it could also be used to print how many
+  // geometries failed. It is mutable to let parsing function which are const
+  // still modify the the nr of failed parsings.
+  mutable size_t nrOfFailedParsedGeometries_ = 0;
```

Comment:
Maybe don't make it mutable and make the parsing functions not const?
The `SpatailJoinAlgorithms` object is something to throw away after a single computation anyway.

and call it `numFailed...` that is more consistent.